README.md

```diff
@@ -22,14 +22,14 @@ Detailed documentation can be found on our [GitHub Documentation Page](http://vm
 
 ## Download
 
-**[Click here to download from Github releases](https://github.com/vmware/docker-volume-vsphere/releases)**
+** Download releases from [Github releases](https://github.com/vmware/docker-volume-vsphere/releases) page**
 
 The download consists of 2 parts:
 
 1. VIB (vDVS driver): The ESX code is packaged as [a vib or an offline depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
 2. Managed plugin (vDVS plugin): Plugin is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
 
-Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/install.html) for vDVS driver's & plugin's installation instruction. Pick the latest release and use the same version of vDVS plugin (on Docker host VM) and driver (on ESX).
+Please check [vDVS Installation User Guide](http://vmware.github.io/docker-volume-vsphere/documentation/install.html) to get started. To ensure compatibility, make sure to use the same version of vDVS driver and managed plugin.
```

Comment:
Though appropriate, I suggest being specific here as follows 
vDVS managed plugin (on Docker host VM) and vDVS driver (on ESX).

This isn't completely intuitive to a person who wants to use this for the first time and is not at all familiar with infrastructure required.

Reply:
agreed!!